### PR TITLE
docs: Documentation for Repeater tag and Multi-page document annotation

### PR DESCRIPTION
**Pull request by anithamuthu2022**
[DEV-3534]

[DEV-3534]: https://heartex.atlassian.net/browse/DEV-3534?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by juliosgarbi — APPROVED** [review]
LGTM

File: deploy/requirements-test.txt [hunk 1]
```diff
@@ -9,4 +9,5 @@ tavern==1.19.0
 fakeredis==1.5.0
 pytest-env==0.6.2
 responses==0.13.0
-pytest-xdist~=2.5.0
\ No newline at end of file
+pytest-xdist~=2.5.0
+importlib-metadata==4.13.0; python_version == '3.7'
```
Comment (makseq) [inline deploy/requirements-test.txt:13]: Why do you add this library line here?

Comment (anithamuthu2022) [reply deploy/requirements-test.txt:13]: I did not add this one. 

Should line #13 be removed?

Comment (nikitabelonogov) [reply deploy/requirements-test.txt:13]: that was added by @triklozoid 
tests were failing on python 3.7

Comment (anithamuthu2022) [reply deploy/requirements-test.txt:13]: @makseq ^^

**Review by makseq — APPROVED** [review]

**Comment by anithamuthu2022** [comment]
https://user-images.githubusercontent.com/107003313/196548891-cc26321c-33fb-49ac-bd8d-df43d1128fb4.mov

**Comment by anithamuthu2022** [comment]
@niklub I've fixed the bug here.

**Review by niklub — APPROVED** [review]

**Comment by niklub** [comment]
Merging with failed tests due to no runtime code included

**Pull request merged**